qa/L0_java_memory_growth/Simple.java

```diff
@@ -394,6 +394,13 @@ output0, output1, new FloatPointer(input0_data), new FloatPointer(input1_data),
       } else {
         return false;
       }
```

Comment:
Don't return early, otherwise the added code will never be reached.

Reply:
Fixed.